src/app/locationInfo/LocationInfo.jsx

```diff
@@ -57,7 +59,11 @@ class LocationInfo extends Component {
   };
 
   render() {
-    const { locationData } = this.props;
+    const { locationData, locationError } = this.props;
+
+    if(locationError){
+      return <ErrorLabel errorMessage={this.props.locationError} />;
```

Comment:
So I guess this will only work for the location form, not service/documents/comments/etc?
Would be nice if we had some "base layout" type thing that handled this, or some way for the router to always recognize there was an error and show it accordingly... but maybe that's difficult.

Reply:
Yes, as I mentioned in my comment above, this will currently only work for locations. Once we decide whether we like the pattern outlined in this PR, I can go through and apply the same pattern to service and documents. Eventually, we will want to generalize these patterns into some reusable action/reducer code.

Reply:
> Right, I think the last part is more what I was referring to.
If we had a "base layout" component that's used for all screens, we wouldn't have to go into all the different pages and change them, whereas the current way seems to introduce a bit of duplication.

Yes, it would definitely be good to start to consolidate this code so that we don't have to reduplicate the "loading screen"/"error screen" logic all over the place.

> I suppose in terms of the Redux side of things, it would require a reducer that listens to error actions in general (either through knowing all the specific ones, or looking for something standardized like an "error" property in the payload), and just puts any errors in a single piece of state. This kind of error reducer is in fact pretty standard, and probably requires less code than the multiple different error reducers. If it's really that simple, and avoids duplication, might be good to just do it that way.

I already do this a bit in the Form component, and the sites that use. You can see this here: 
https://github.com/streetlives/streetlives-web/pull/112/files#diff-a70d9e4934a427002e689cb0166f5d22R61

Reply:
Right, the Form example is great.

I was thinking that this might be a good chance to do that consolidation, seeing as you already need to touch all these sites anyway (so we could just as easily make them all use that core layout instead of making them use ErrorLabel specifically).

But since you've already written all this, if you don't feel like changing it, I suppose we can do it some other time.

Reply:
I created an issue for this: https://github.com/streetlives/streetlives-web/issues/124